service/version_bundles.go

```diff
@@ -3,9 +3,11 @@ package service
 import (
 	"github.com/giantswarm/versionbundle"
 
-	"github.com/giantswarm/aws-operator/service/controller/v22"
-	"github.com/giantswarm/aws-operator/service/controller/v23"
-	"github.com/giantswarm/aws-operator/service/controller/v24"
+	v22 "github.com/giantswarm/aws-operator/service/controller/v22"
+	v22patch1 "github.com/giantswarm/aws-operator/service/controller/v22patch1"
+	v23 "github.com/giantswarm/aws-operator/service/controller/v23"
+	v23patch1 "github.com/giantswarm/aws-operator/service/controller/v23patch1"
+	v24 "github.com/giantswarm/aws-operator/service/controller/v24"
```

Comment:
Let's please remove the aliases here and in the other places.

Reply:
I visual code stuff

Reply:
Removed aliases(I need to know why why go plugin in vcode adds them)